src/views/Entries/QualityControl/EntriesStats/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useMemo } from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import { EntrySummary } from '#typings/entry';
+import ListView from '#rscv/List/ListView';
+import _ts from '#ts';
+
+import styles from './styles.css';
+
+interface Stats {
+    id: string;
+    title: string;
+    value?: number;
+}
+
+type StaticEntrySummary = Omit<EntrySummary, 'orgTypeCount'>;
+
+const staticEntryStatTitles: { [ key in (keyof StaticEntrySummary)]: string } = {
+    totalLeads: _ts('entries.qualityControl', 'totalLeads'),
+    totalSources: _ts('entries.qualityControl', 'totalSources'),
+    totalUnverifiedEntries: _ts('entries.qualityControl', 'totalUnverifiedEntries'),
+    totalVerifiedEntries: _ts('entries.qualityControl', 'totalVerifiedEntries'),
+};
+
+interface EntryStatProps {
+    title: string;
+    value: number;
+    max: number;
+    className?: string;
+}
+
+function EntryStat({
+    title,
+    value = 0,
+    max,
```

Comment:
Should we check if max is zero? before division?

Reply:
No its fine